lib/executables/validate.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable no-use-before-define */
+const assertion = require('assert');
+const fs = require('fs');
+const path = require('path');
+const utils = require('../helpers/utils');
+const { print } = require('../helpers/log');
+
+const VIEW_CLASSES = [
+  'TextFieldView', 'TextAreaView', 'CheckBoxView', 'SelectView', 'MultiSelectView', 'PasswordFieldView', 'OAuthFieldView',
+];
```

Comment:
`Set` is more idiomatic

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set